docs/index.asciidoc

```diff
@@ -23,14 +23,13 @@ include::{include_path}/plugin_header.asciidoc[]
 
 Periodically run a shell command and capture the whole output as an event.
 
-Notes:
-
+[NOTE]
+========
 * The `command` field of this event will be the command run.
 * The `message` field of this event will be the entire stdout of the command.
+========
 
-===== IMPORTANT
-
-The exec input ultimately uses `fork` to spawn a child process.
+IMPORTANT: The exec input ultimately uses `fork` to spawn a child process.
```

Comment:
Thank you for getting this extra changes in.  They render nicely and look good. Post-merge LGTM